utils/create_fhir_store.sh

```diff
@@ -22,7 +22,8 @@ if [[ $# -ne 4 ]]; then
   exit 1
 fi
 
-# TODO create GCP Healthcare data-set too instead of assuming it exists.
+# Create GCP Healthcare dataset if it does not exist
+gcloud healthcare datasets create ${3} --location=${2}
```

Comment:
Thanks for adding this. Does this still succeeds when the dataset is already created?

Reply:
Yes, it does. It prints `ERROR: (gcloud.healthcare.datasets.create) ALREADY_EXISTS: already exists` and continues the script execution.  If we want to avoid that, maybe we can do something like in the code below. The BQ create dataset command also does the same thing.  Maybe we could try making the dataset creation into a function?

```
output=$(gcloud healthcare datasets create ${3} --location=${2} 2>&1)
code=$?

if [[ $code -eq 0 ]]; then
    echo 'Created dataset'
else   
  if [[ $output == *'already exists'* ]]; then
        echo 'Dataset already exists'
  else
        echo $output
        exit -1
  fi
fi
```